applications/fake-services/app/templates/minio-ui-deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: minio-ui-setup
+  labels:
+    {{- include "fake-service.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "fake-service.minioui.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+      {{- include "fake-service.minioui.selectorLabels" . | nindent 8 }}
+    spec:
+      initContainers:
+        - name: setup-minio-ui
+          image: minio/minio
+          env:
+            - name: MINIO_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: kotsadm-minio
+                  key: accesskey
+            - name: MINIO_SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: kotsadm-minio
+                  key: secretkey
+          command:
+            - /bin/sh
+            - -c
+            - |
+              mc alias set myminio http://kotsadm-minio:9000 $MINIO_ACCESS_KEY $MINIO_SECRET_KEY
+              echo "Creating Minio Console Admin User..."
+              mc admin user add myminio admin $MINIO_SECRET_KEY 
+              echo "Attaching IAM policy..."
+              mc admin policy attach myminio consoleAdmin --user admin
+      containers:
+        - name: minio-ui
+          image: alpine
+          command:
+            - /bin/sh
+            - -c
+            - |
+              echo "Check WebUI port in kotsadm-minio pod logs and port forward to it to browse the Minio UI"
+              echo "e.g. kubectl port-forward svc/kotsadm-minio 33655:33655"
+              echo "username: admin, password: set to secretkey in kotsadm-minio secret"
+              sleep infinity
+      restartPolicy: Always
```

Comment:
I'm somewhat confused by this. Should this Minio UI setup just be a job, or an `initContainer` on `minio` specifically?

Reply:
This pod serves as a temporary helper to provide MinIO UI access instructions. As Minio pod used in KOTS already provided UI, we added this setup to make it easier for CRE to know how to access that UI for troubleshooting purposes. It's not essential for the app , more of a developer convenience for us.

Reply:
@nvanthao curious to know more of the workflow in how a Vendor is going to see these instructions though. They will need to check the logs on the pod right? Why wouldn't something like this instead be in the NOTES.txt or something like that?

Reply:
It's just convenient for me since the pod will also setup the Minio Admin user in `initContainer`, but great idea in the NOTES.txt, let me update the README on this.